src/cmd/common/common.go

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package common handles command configuration across all commands
+package common
+
+import (
+	"github.com/defenseunicorns/zarf/src/types"
+)
+
+// SetBaseDirectory sets base directory on package config when given in args
+func SetBaseDirectory(args []string, pkgConfig *types.PackagerConfig) {
+	if len(args) > 0 {
+		pkgConfig.CreateOpts.BaseDir = args[0]
+	} else {
+		pkgConfig.CreateOpts.BaseDir = "."
```

Comment:
The reason I changed it to "." instead of os.Getwd() is because `zarf prepare lint` without a path would print out the full file system path. We could also argue that's something we want, I can see both ways. AFAIK this won't affect package create.

Reply:
Discussed with team, we feel this is fine